setup/create_db.pl

```diff
@@ -387,12 +390,17 @@ sub add_postgresveximuser {
 #####################################
 
 sub add_siteadminuser {
- 
+ my $crypted;
  	if($act eq  "newdb")
  	{
 	  if ($dbtype eq "mysql") { $dbh = $mydbh; } 
 	#elsif ($dbtype == "pgsql") { $dbh = $pgdbh; }
-	
+        print "Select an hashing scheme for the user passwords:\n";
+        print "sha512:\tcompatible with Dovecot/Courier\n";
+        print "bcrypt:\tcompatible only in *BSD with Dovecot/Courier (do not use for Linux)\n";
+        print "clear:\tcompatible with Dovecot/Courier, not recommended\n";
+        print "\nPlease select a hashing scheme (default: sha512): ";
+        chomp($hashmethod = <STDIN>);
```

Comment:
I noticed after my pull request and after I spent some time to get this working in perl ;-( Anyway, it would make more sense to make these installation-scripts and migration steps in php as well (for the future).